<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Access_Control</class>
 <widget class="QMainWindow" name="Access_Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="newWindow">
   <property name="styleSheet">
    <string notr="true">*{
	border:1px solid whilte;
	border-radius:5px;
}
#newWindow,#consoleEdit{
	background-color: #161f2b;
}
QCheckBox, QPushButton{
	padding:2px 5px;
}
QLineEdit{background-color: rgba(0,0,0,0);
	border:none;
	border-bottom:2px solid rgba(46,82,101,200);
	color: rgb(214, 214, 214);
	padding-bottom:7px;
}
#buttons,#input_frame{
	border:none;}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="InputFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="InputLabel">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Test Cases and User Input</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="input_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <widget class="QLineEdit" name="input_02">
            <property name="placeholderText">
             <string>Sudo Password</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="input_03">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>NMS Username</string>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <widget class="QLineEdit" name="input_06">
            <property name="placeholderText">
             <string>fm-pm password</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="input_01">
            <property name="placeholderText">
             <string>Sudo Username</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLineEdit" name="input_04">
            <property name="placeholderText">
             <string>NMS Password</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="input_09">
            <property name="placeholderText">
             <string>RU Fronthaul Interface Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="9">
           <widget class="QLineEdit" name="input_12">
            <property name="placeholderText">
             <string>RX Arfcn</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="input_07">
            <property name="placeholderText">
             <string>Paragon IP</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="input_08">
            <property name="placeholderText">
             <string>PTP &amp; SYNCE Port</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLineEdit" name="input_10">
            <property name="placeholderText">
             <string>Bandwidth</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="input_13">
            <property name="placeholderText">
             <string>TX Center Frequency</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="input_14">
            <property name="placeholderText">
             <string>RX Center Frequency</string>
            </property>
           </widget>
          </item>
          <item row="2" column="9">
           <widget class="QLineEdit" name="input_17">
            <property name="placeholderText">
             <string>Currupt FIle Path</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="input_15">
            <property name="placeholderText">
             <string>DU Password</string>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QLineEdit" name="input_11">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>TX Arfcn</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLineEdit" name="input_05">
            <property name="placeholderText">
             <string>fm-pm username</string>
            </property>
           </widget>
          </item>
          <item row="2" column="6" colspan="2">
           <widget class="QLineEdit" name="input_16">
            <property name="placeholderText">
             <string>Software File Path</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="buttons">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="submitBtn">
            <property name="text">
             <string>Submit</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/icons/zap.svg</normaloff>:/icons/icons/zap.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="OutputFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignTop">
        <widget class="QLabel" name="outputLabel">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Output</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="consoleEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
